.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!--
+  - Thanks for submitting code to Oppiabot! Please fill out the following as part of
+  - your pull request so we can review your code more easily.
+  -->
+
+## Explanation
+<!--
+  - Explain what your PR does. If this PR fixes an existing bug, please include
+  - "Fixes #bugnum:" in the explanation so that GitHub can auto-close the issue
+  - when this PR is merged.
+  -->
+
+## Checklist
+- [ ] I have successfully deployed my own instance of Oppiabot.
+  - If you're unaware of the deployment process, please follow the instructions given [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
```

Comment:
I think you can skip the "if you're unaware" bit and just write:

```You can find instructions for doing this at ...```

Reply:
Done, ptal!